vendor/visualmetrics.py

```diff
@@ -1457,6 +1462,41 @@ def calculate_speed_index(progress):
         last_progress = p['progress'] / 100.0
     return int(si)
 
+def calculate_contentful_speed_index(progress, directory):
+    x = len(progress)
+    dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), directory)
+    first_paint_frame = os.path.join(
+        dir, "ms_{0:06d}.png".format(
+            progress[1]["time"]))
```

Comment:
Indexing starts at 0 in Python.  Do you really want to skip `progress[0]`?  Your subsequent loop runs from `1:`, so you're just dropping the first frame here.